cmd/gapit/flags.go

```diff
@@ -366,15 +366,16 @@ type (
 		ADB         string         `help:"Path to the adb executable; leave empty to search the environment"`
 	}
 	ScreenshotFlags struct {
-		Gapis      GapisFlags
-		Gapir      GapirFlags
-		At         []flags.U64Slice `help:"command/subcommand index (e.g. '[123, 0, 0, 4]') for the screenshot (repeatable)"`
-		Frame      []int            `help:"frame index for the screenshot (repeatable). Empty for last"`
-		Draws      bool             `help:"create a screenshot of every draw call in the requested frame(s) (only honored if using -frame)"`
-		Out        string           `help:"output image file (default 'screenshot.png')"`
-		NoOpt      bool             `help:"disables optimization of the replay stream"`
-		Attachment string           `help:"the attachment to show (0-3 for color, d for depth, s for stencil)"`
-		Overdraw   bool             `help:"renders the overdraw instead of the color framebuffer"`
+		Gapis          GapisFlags
+		Gapir          GapirFlags
+		At             []flags.U64Slice `help:"command/subcommand index (e.g. '[123, 0, 0, 4]') for the screenshot (repeatable)"`
+		Frame          []int            `help:"frame index for the screenshot (repeatable). Empty for last"`
+		Draws          bool             `help:"create a screenshot of every draw call in the requested frame(s) (only honored if using -frame)"`
+		ExecutedDraws  int				`help:"create a screenshot of evenly spaced executed draw calls. Number provided is the maximum amount. (not used with -frame)"`
```

Comment:
Broken indentation.

I think we may have a help text maybe a bit more explicit:

```
create a screenshot at every (total-draw-calls / this-flag-value) drawcall index in the frame, e.g. for a frame with 100 draw calls, '-executed-draws 4' takes a screenshot at every 25 draw calls.
```